tests/integration/_cases/send_event/send_event-raw.trycmd

```diff
@@ -13,8 +13,9 @@ $ sentry-cli send-event --log-level=debug
 > --fingerprint custom-fingerprint
 > --no-environ
 ? success
-  DEBUG   [..] sentry-cli version: [VERSION], platform: [..], architecture: [..]
-  INFO    [..] sentry-cli was invoked with the following command line: "[CWD]/target/debug/sentry-cli[EXE]" "send-event" "--log-level=debug" "--level" "debug" "--timestamp" "1649335000929" "--release" "my-release" "--dist" "my-dist" "--env" "production" "--message" "hello" "--platform" "prod" "--tag" "hello:there" "--extra" "hello:there" "--user" "id:42" "--fingerprint" "custom-fingerprint" "--no-environ"
+  INFO    [..] Loaded config from [CWD]/.sentryclirc
+  DEBUG   [..] sentry-cli version: [VERSION], platform: "darwin", architecture: "arm64"
```

Comment:
you should still mask the platform and architecture for these tests, they are failing for me as I am still on `x86_64`.